integreat_cms/static/src/js/analytics/statistics-page-accesses.ts

```diff
@@ -176,7 +175,6 @@ export const setPageAccessesEventListeners = () => {
     statisticsForm = document.getElementById("statistics-form") as HTMLFormElement;
     pageAccessesForm = document.getElementById("statistics-page-access") as HTMLFormElement;
     pageAccessesURL = pageAccessesForm.getAttribute("data-page-accesses-url");
```

Comment:
On all pages except for the statistics pages the `pageAccessesForm` element doesn't exist and throws an error in the browser terminal. Do you think we could add a line like this? :)
```suggestion
    if (pageAccessesForm) {
      pageAccessesURL = pageAccessesForm.getAttribute("data-page-accesses-url");
```